### DISCUSSION TITLE
I have a problem implementing Geckolib in my project in IntelliJ, Fabric 1.20.4.

### DISCUSSION THREAD

**Discussion by TheWayRaw** [bernie-g/geckolib#608] [General]
I have a problem implementing Geckolib in my project in IntelliJ, Fabric 1.20.4.

According to the documentation (https://github.com/bernie-g/geckolib/wiki/Installation-(Geckolib4))

repositories {
  maven {
          name = 'GeckoLib'
          url 'https://dl.cloudsmith.io/public/geckolib3/geckolib/maven/'
          content {
                  includeGroupByRegex("software\\.bernie.*")
                  includeGroup("com.eliotlash.mclib")
          }
  }
}

dependencies {
        modImplementation("software.bernie.geckolib:geckolib-fabric-${minecraft_version}:${geckolib_version}")
        implementation("com.eliotlash.mclib:mclib:20")
}

And:
gradle.properties:  ->  geckolib_version=4.4.6

But it gives me this error:

A problem occurred configuring root project 'Arcanum'.
> Could not resolve all files for configuration ':modImplementation'.
   > Could not find software.bernie.geckolib:geckolib-fabric-1.20.4:4.4.6.
     Searched in the following locations:
       - file:/C:/Users/Anto/Documents/ServerPruebasMinecrat/Servers%20Fabric/Proyectos%20Mods/Arcanum/.gradle/loom-cache/remapped_mods/software/bernie/geckolib/geckolib-fabric-1.20.4/4.4.6/geckolib-fabric-1.20.4-4.4.6.pom
       - file:/C:/Users/Anto/.gradle/caches/fabric-loom/minecraftMaven/software/bernie/geckolib/geckolib-fabric-1.20.4/4.4.6/geckolib-fabric-1.20.4-4.4.6.pom
       - file:/C:/Users/Anto/Documents/ServerPruebasMinecrat/Servers%20Fabric/Proyectos%20Mods/Arcanum/.gradle/loom-cache/minecraftMaven/software/bernie/geckolib/geckolib-fabric-1.20.4/4.4.6/geckolib-fabric-1.20.4-4.4.6.pom
       - https://maven.fabricmc.net/software/bernie/geckolib/geckolib-fabric-1.20.4/4.4.6/geckolib-fabric-1.20.4-4.4.6.pom
       - https://libraries.minecraft.net/software/bernie/geckolib/geckolib-fabric-1.20.4/4.4.6/geckolib-fabric-1.20.4-4.4.6.pom
       - https://libraries.minecraft.net/software/bernie/geckolib/geckolib-fabric-1.20.4/4.4.6/geckolib-fabric-1.20.4-4.4.6.jar
       - https://repo.maven.apache.org/maven2/software/bernie/geckolib/geckolib-fabric-1.20.4/4.4.6/geckolib-fabric-1.20.4-4.4.6.jar
       - https://repo.maven.apache.org/maven2/software/bernie/geckolib/geckolib-fabric-1.20.4/4.4.6/geckolib-fabric-1.20.4-4.4.6.pom
       - https://dl.cloudsmith.io/public/geckolib3/geckolib/maven/software/bernie/geckolib/geckolib-fabric-1.20.4/4.4.6/geckolib-fabric-1.20.4-4.4.6.pom
     Required by:
         project :

Possible solution:
 - Declare repository providing the artifact, see the documentation at https://docs.gradle.org/current/userguide/declaring_repositories.html

**Comment by Tslat** [comment]
A few things
1. Geckolib support is done in the Discord server, not github
2. 4.4.6 isn't a version for 1.20.4
3. 1.20.4 is a dead version and you shouldn't be modding on it, use 1.21

**Reply by TheWayRaw** [reply]
So, there's nothing to do in version 1.20.4? Geckolib doesn't work for this version?

**Comment by ShrinkShi** [comment]
Similarly, my 1.20.1 version also had the same problem. I clicked on the url 'https://dl.cloudsmith.io/public/geckolib3/geckolib/maven/' and found that even with the proxy, it still showed 404 not found.So I have reason to suspect that the Gecko library is invalid? Or some other reason.

**Comment by Tslat** [comment]
Works fine

Again, Geckolib support is done on the discord server, not here
